src/client/components/TestComponent/MealList.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+
+function MealList() {
+  const [meals, setMeals] = useState([]);
+
+  useEffect(() => {
+    async function fetchMeals() {
```

Comment:
It would be more standard to have the function definition outside of useEffect, it you want this to be a separate function. 
It could also just be the statements performed right away in useEffect 💡

Reply:
I'll do that. thank you for the feedback :) I really appreciated it :)